app/code/Magento/MediaGalleryIntegration/composer.json

```diff
@@ -6,7 +6,8 @@
         "magento/framework": "*",
         "magento/module-media-gallery-ui-api": "*",
         "magento/module-media-gallery-api": "*",
-        "magento/module-media-gallery-synchronization-api": "*"
+        "magento/module-media-gallery-synchronization-api": "*",
+        "magento/module-ui": "*"
```

Comment:
Please create a pull request to https://github.com/magento/partners-magento2-infrastructure with an update to composer etalon (see https://github.com/magento/partners-magento2-infrastructure/pull/19 for example)